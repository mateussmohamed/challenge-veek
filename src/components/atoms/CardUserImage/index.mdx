---
name: CardUserImage
menu: Atoms
---

import { Playground, PropsTable } from 'docz'
import CardUserImage from './index'

# CardUserImage
<PropsTable of={CardUserImage} />

<Playground>
  <CardUserImage />
</Playground>
