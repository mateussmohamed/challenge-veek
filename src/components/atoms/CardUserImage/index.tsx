import styled from 'styled-components'

export const CardUserImage = styled.div`
  width: 27px;
  height: 27px;
  display: inline-block;
  background-color: grey;
  border-radius: 100%;
  margin-right: 15px;
`
