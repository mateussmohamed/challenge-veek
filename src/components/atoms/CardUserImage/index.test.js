import React from 'react'
import { shallow } from 'enzyme'
import 'jest-styled-components'

import CardUserImage from './index'

describe('[<CardUserImage />]', () => {
  const wrap = (props = {}) => shallow(<CardUserImage {...props} />)

  test('matches to snapshot', () => {
    const wrapper = wrap()

    expect(wrapper).toMatchSnapshot()
  })
})
