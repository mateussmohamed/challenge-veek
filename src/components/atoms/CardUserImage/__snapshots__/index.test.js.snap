// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[<CardUserImage />] matches to snapshot 1`] = `
<StyledComponent
  forwardedClass={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": undefined,
      "componentStyle": ComponentStyle {
        "componentId": "CardUserImage-sc-6l1fwq-0",
        "isStatic": true,
        "rules": Array [
          "width:27px;height:27px;display:inline-block;background-color:grey;border-radius:100%;margin-right:15px;",
        ],
      },
      "displayName": "CardUserImage",
      "render": [Function],
      "styledComponentId": "CardUserImage-sc-6l1fwq-0",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
